api/v1alpha1/webhook_suite_test.go

```diff
@@ -141,335 +141,316 @@ var _ = AfterEach(func() {
 })
 
 var _ = Describe("Webhook Tests", func() {
-	Context("Functional Tests", func() {
```

Comment:
most of these changes are coming because we have reverted the webhook txn?
and also because there is a new Database object in the ndb crd?

Reply:
No, should've added a comment explaining these changes.
So basically the `Describe` and `It` blocks were redundant in these testcases so I removed them. Simply put, all the tests in this file were running in a separate context/suite. Now they run in a single suite.